libevm/VMOpt.cpp

```diff
@@ -63,14 +63,16 @@ void VM::copyCode(int _extraBytes)
 
 void VM::optimize()
 {
+	copyCode(33);
```

Comment:
Due to line 112 we might have to copy much more. Perhaps we can just append the zeros after the first pass?